tests/test_polydata.py

```diff
@@ -274,61 +274,72 @@ def test_edge_mask(sphere):
     mask = sphere.edge_mask(10)
 
 
-def test_boolean_cut_inplace(sphere, sphere_shifted):
-    sub_mesh = sphere
-    sub_mesh.boolean_cut(sphere_shifted, inplace=True)
-    assert sub_mesh.n_points
-    assert sub_mesh.n_cells
+def test_boolean_union(sphere, sphere_shifted):
+    union = sphere.boolean_union(sphere_shifted)
+    intersection = sphere.boolean_intersection(sphere_shifted)
 
+    # union is volume of sphere + sphere_shifted minus the part intersecting
+    expected_volume = sphere.volume + sphere_shifted.volume - intersection.volume
+    assert np.isclose(union.volume, expected_volume, atol=1E-3)
 
-def test_boolean_cut_fail(plane):
-    with pytest.raises(NotAllTrianglesError):
-        plane - plane
 
+def test_boolean_intersection(sphere, sphere_shifted):
+    intersection = sphere.boolean_intersection(sphere_shifted)
+    union = sphere.boolean_union(sphere_shifted)
```

Comment:
These two are basically the same test so I think we should join them into something like `test_boolean_union_intersection`.

Reply:
done in 9405472c6